src/index.ts

```diff
@@ -67,14 +67,4 @@ export type { TextFieldProps } from "src/form/TextField/TextField";
 export { default as Upload } from "src/form/Upload/Upload";
 export type { UploadProps } from "src/form/Upload/Upload";
 
-export {
-  heading1,
-  heading2,
-  heading3,
-  heading4,
-  heading5,
-  heading6,
-  body1,
-  body2,
-  narrow,
-} from "src/guidelines/theme";
+export * from "src/guidelines/theme";
```

Comment:
There is something I missing.
In src/guidelines/theme/index.ts, you export typographies
Here you export everything from theme
But in files like src/navigation/Breadcrumb/Breadcrumb.tsx you import body from typographies file and not from theme/index file
I don't understand something or I missed something ?

Reply:
Both would work, actually.
I think the index files are mainly used for exporting (to the project using purrfect-ui), while you still can use direct imports from within purrfect-ui

Reply:
ok